template_helper.go

```diff
@@ -113,3 +117,10 @@ func regexFindFirstSubmatch(regex string, s string) string {
 	}
 	return matches[1]
 }
+
+// hmacv1 computes SHA256 HMAC of data using secret
```

Comment:
can we rename it to `hmacSHA256`? v1 is not clear what hash function is used

Reply:
Done